src/CommandHandler.cxx

```diff
@@ -7,12 +7,12 @@ namespace FileWriter {
 // In the future, want to handle many, but not right now.
 static int g_N_HANDLED = 0;
 
-CommandHandler::CommandHandler(Master *master,
-                               rapidjson::Value const *config_file)
-    : master(master), config_file(config_file) {
+CommandHandler::CommandHandler(MainOpt &config, Master *master)
+    : config(config), master(master) {
+  // Will take care of this in upcoming PR.
```

Comment:
Consider making issues in github rather than leaving comments in the code. Then it is documented in a way which can be referenced in the PR and is harder to forget about.